docs/src/manual/complex.md

```diff
@@ -21,7 +21,7 @@ Create a complex-valued variable using [`ComplexPlane`](@ref):
 julia> model = Model();
 
 julia> @variable(model, x in ComplexPlane())
-real(x) + (0.0 + 1.0im) imag(x)
+real(x) + im imag(x)
```

Comment:
I don't know if I like this. Wouldn't `real(x) + imag(x) im` make more sense?

Reply:
Yes, that would look better, good point